ci/unit/test_nlt_node.sh

```diff
@@ -21,7 +21,7 @@ sudo bash -c ". ./utils/sl/setup_local.sh; ./utils/setup_daos_server_helper.sh"
 
 # NLT will mount /mnt/daos itself.
 # TODO: Enable this for DAOS-10905
-# ./utils/node_local_test.py --max-log-size 800MiB --dfuse-dir /localhome/jenkins/ \
+# ./utils/node_local_test.py --max-log-size 1500MiB --dfuse-dir /localhome/jenkins/ \
 #			   --server-valgrind all
 
-./utils/node_local_test.py --max-log-size 900MiB --dfuse-dir /localhome/jenkins/ all
+./utils/node_local_test.py --max-log-size 1500MiB --dfuse-dir /localhome/jenkins/ all
```

Comment:
I would prefer not to make so big a change here for a couple of reasons.  Parsing the log-file takes some time so this will extend the test time in CI, but also most of NLT is unit testing on the client so will hit various server code paths but typically not all that many times.

NLT will start the server twice, initially with full debugging on where it runs client unit tests, then it will re-start it without debugging to test restart and at that point it runs client-heavy testing.

If this branch causes the server logfile to be so much bigger then the test should be moved after the server restart.

Reply:
the simple matter of the fact here is that the mwc adds a lot of server side debug logs. more tests = more logs.
so there is no way around this here.

Reply:
again if this needs to be changed, we can do that later.. those tests were already approved and landed to the feature branch.

Reply:
> If this branch causes the server logfile to be so much bigger then the test should be moved after the server restart.

IMO just rearranging a test because of server side log files size is large is just a bad approach. ideally every NLT test should have its own logs, so that is not a good solution to the issue. 1g is already large enough to parse. 2g or 3g is not going to be different..

Reply:
Overall this change adds a couple of minutes to NLT but that's probably something we have to live with.  I didn't realise this PR had server side changes.

Most tests are run in parallel so you can't distinguish the server logs between tests, the pydaos tests however are run in serial, partly for historic reasons but as they also call daos_init() it would be a challenge to make them run in parallel and maintain any ability to debug the test.

The log size with this PR is reported as 1.3GiB so this is reasonable if you don't want to run this new test without server debug and think the increased run-time is worth it.